.circleci/check-testnet-deployment.js

```diff
@@ -51,8 +67,13 @@ async function eventuallyClosingBlocks({ chainId, nodes }) {
 }
 
 (async () => {
-    const nodes = topology.network;
-    const chains = topology.chains.map(chain => chain.Id);
+    const nodes = topology.network.filter(({ ip }) => ip !== '54.149.67.22');
```

Comment:
Why is there a special case for this specific IP? It should be configurable.

Reply:
I preferably want to remove this filtering all together.
For how long is this node going to be out of commission?

Reply:
We can redeploy it tomorrow, doesn't really matter to me anymore.